### PR TITLE
liability weight now accept number with , spaces and words at the begi…

### PR DESCRIPTION
**Pull request by mauro-fanton**
…ning and end

### Description of Work carried through

Brief description of what/why/how.

### CODE REVIEW

File: app/forms/liability/LiabilityWeight.scala [hunk 1]
```diff
@@ -70,4 +70,15 @@ object LiabilityWeight {
   def toForm(liabilityWeight: LiabilityWeight): Option[Option[String]] =
     Some(liabilityWeight.totalKg.map(_.toString))
 
+  def stripSpacesAndLetters(weight: String) = {
+    val extractNumberRegex = """^[^\d-]*(-?\d+\.?\d*)\D*$""".r
```
Comment (lmcewanhmrc) [inline app/forms/liability/LiabilityWeight.scala:74]: what's the point of the ^[^\d-]* part?

File: test/forms/liability/LiabilityWeightTest.scala [hunk 2]
```diff
@@ -60,6 +60,27 @@ class LiabilityWeightTest extends AnyWordSpec with Matchers {
...
+        val form = LiabilityWeight.form().bind(input)
+        form.errors.size mustBe 0
+      }
+
+      "amount contain a words" in {
+        val input = Map(totalKg -> "ghkghkj10125hjghjk")
+
+        val form = LiabilityWeight.form().bind(input)
+        form.errors.size mustBe 0
+      }
```
Comment (lmcewanhmrc) [inline test/forms/liability/LiabilityWeightTest.scala:83]: Do we test for extracting the correct values somewhere? 🌮

File: app/forms/liability/LiabilityWeight.scala (hunk 1 above)
Comment (mauro-fanton) [reply app/forms/liability/LiabilityWeight.scala:74]: I guess it is for not matching such number 123YUG124

File: test/forms/liability/LiabilityWeightTest.scala (hunk 2 above)
Comment (mauro-fanton) [reply test/forms/liability/LiabilityWeightTest.scala:83]: yep we could do that into that test actually

Comment (mauro-fanton) [reply test/forms/liability/LiabilityWeightTest.scala:86]: done

**Review by lmcewanhmrc — APPROVED** [review]

**Review by PSouthern — APPROVED** [review]

**Pull request merged**